<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Alpha-7VTW322" failures="0" tests="6" name="com.MyTests.LoginTest" time="165.062" errors="2" timestamp="24 Jun 2020 00:56:13 GMT" skipped="0">
  <testcase classname="com.MyTests.LoginTest" name="verifyLoginPageTitleTest" time="16.318"/>
  <testcase classname="com.MyTests.LoginTest" name="verifyLoginPageTitleTest" time="31.632"/>
  <testcase classname="com.MyTests.LoginTest" name="verifyLoginPageHeaderTest" time="15.524"/>
  <testcase classname="com.MyTests.LoginTest" name="verifyLoginPageHeaderTest" time="23.219"/>
  <testcase classname="com.MyTests.LoginTest" name="doLoginTest" time="48.750">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.MyPages.BasePage.getPageHeader(BasePage.java:22)
at com.MyPages.HomePage.getHomePageHeader(HomePage.java:32)
at com.MyTests.LoginTest.doLoginTest(LoginTest.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </error>
  </testcase> <!-- doLoginTest -->
  <testcase classname="com.MyTests.LoginTest" name="doLoginTest" time="29.619">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.MyPages.BasePage.getPageHeader(BasePage.java:22)
at com.MyPages.HomePage.getHomePageHeader(HomePage.java:32)
at com.MyTests.LoginTest.doLoginTest(LoginTest.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </error>
  </testcase> <!-- doLoginTest -->
</testsuite> <!-- com.MyTests.LoginTest -->
